acct/check

```diff
@@ -132,11 +136,57 @@ if [ -f "$vhost_file" ] && vhost=$(grep -E "^[#[:space:]]*$user[![:space:]]" "$v
 fi
 
 # check apphosting config
-if [ -f "$vhost_app_file" ] && vhost=$(grep -E "^[#[:space:]]*$user[![:space:]]" "$vhost_app_file"); then
+if [ -f "$vhost_app_file" ] && vhost_app=$(grep -E "^[#[:space:]]*$user[![:space:]]" "$vhost_app_file"); then
   echo
   echo "Virtual apphosts:"
   echo "-------------------------------------------------------------------------------"
-  echo "$vhost"
+  echo "$vhost_app"
+  echo "-------------------------------------------------------------------------------"
+fi
+
+# check virtual mail config
+if [ -f "$vhost_mail_file" ] && vhost_mail=$(grep -E "^[#[:space:]]*$user[![:space:]]" "$vhost_mail_file"); then
+  echo
+  echo "Virtual mail:"
+  echo "-------------------------------------------------------------------------------"
+  echo "$vhost_mail"
+  echo "-------------------------------------------------------------------------------"
+fi
+
+
+# check dns entries
+expand_fqdns()
+{
+  while read -r entry; do
+    if [ -z "$entry" ]; then
+      continue
+    fi
+    
+    username=$(echo "$entry" | cut -d' ' -f1)
+    domain=$(echo "$entry" | cut -d' ' -f2)
+
+    # shellcheck disable=SC2143,SC2086,SC2063
+    if [ "$domain" = "-" ]; then
+      echo "$username.berkeley.edu"
+    elif [ "$(echo $domain | grep -E '*\.[a-z]+$')" ]; then
```

Comment:
To fix [SC2143](https://github.com/koalaman/shellcheck/wiki/SC2143), [SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086), and [SC2063](https://github.com/koalaman/shellcheck/wiki/SC2063), this should just be:

```
elif echo "$domain" | grep -qE '.+\.[a-z]+$'; then
```

Also, if the goal here is to just check if the domain contains a dot, then you can make it even simpler and just do something like `grep -q '\.'` instead.

Reply:
I had disabled this check because using `grep -qE` was causing this clause to trigger incorrectly in certain cases but it turns out it was just bad bash on my part. This is fixed now and no more shellcheck problems.